src/qunit.css

```diff
@@ -68,6 +68,12 @@
 	overflow: hidden;
 }
 
+#qunit-filteredTest {
+	padding: 0.5em 1em 0.5em 1em;
+	background-color: #F4FF77;
+	color: #366097;
```

Comment:
Not sure what else might look good either, but I'd be happy to take suggestions.